### ISSUE TITLE
qTox won't open in OS X, Yosemite

### ISSUE THREAD

**Issue by cmacfergus** [qTox/qTox#1507]
I am using version 10.10.2, and every time I attempt to open qTox, this is the error message I get:

Process:               qtox [17518]
Path:                  /Applications/qtox.app/Contents/MacOS/qtox
Identifier:            im.tox.qtox
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           qtox [17518]
User ID:               501

Date/Time:             2015-04-06 15:15:13.248 -0700
OS Version:            Mac OS X 10.10.2 (14C1514)
Report Version:        11
Anonymous UUID:        905D6910-490E-A76E-1E4F-BA1F256A8456

Sleep/Wake UUID:       386822B1-6D29-45CD-A9E9-A384DDD2ECA1

Time Awake Since Boot: 10000 seconds
Time Since Wake:       510 seconds

Crashed Thread:        Unknown

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000002019

Backtrace not available

Unknown thread crashed with X86 Thread State (64-bit):
  rax: 0x0000000000002000  rbx: 0x00007fff5fc3f860  rcx: 0x000000000000000d  rdx: 0x00007fff5fbfe9c0
  rdi: 0x00007fff5fc3f860  rsi: 0x0000000000000001  rbp: 0x00007fff5fbfe9c0  rsp: 0x00007fff5fbfe9c0
   r8: 0x00007fff5fc23aec   r9: 0x00007fff5fbfe760  r10: 0x0000000100000000  r11: 0x00007fff5fc40460
  r12: 0x00007fff5fc27920  r13: 0x00007fff5fbfea10  r14: 0x00007fff5fc3f860  r15: 0x00007fff5fc40ea0
  rip: 0x00007fff5fc11644  rfl: 0x0000000000010206  cr2: 0x0000000000002019

Logical CPU:     0
Error Code:      0x00000004
Trap Number:     14

Binary images description not available

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 20073
    thread_create: 0
    thread_set_state: 0

Model: iMac7,1, BootROM IM71.007A.B03, 2 processors, Intel Core 2 Duo, 2.8 GHz, 4 GB, SMC 1.21f4
Graphics: ATI Radeon HD 2600 Pro, ATI,RadeonHD2600, PCIe, 256 MB
Memory Module: BANK 0/DIMM0, 2 GB, DDR2 SDRAM, 667 MHz, 0x7F4F000000000000, 0x4A4D3636375153552D324720202020202020
Memory Module: BANK 1/DIMM1, 2 GB, DDR2 SDRAM, 667 MHz, 0x7F4F000000000000, 0x4A4D3636375153552D324720202020202020
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x88), Broadcom BCM43xx 1.0 (5.10.131.36.16)
Bluetooth: Version 4.3.2f6 15235, 3 services, 26 devices, 1 incoming serial ports
Network Service: AirPort, AirPort, en1
Serial ATA Device: ST3500630AS Q, 500.11 GB
Parallel ATA Device: OPTIARC  DVD RW AD-5630A
USB Device: DataTraveler 2.0
USB Device: Built-in iSight
USB Device: Cruzer
USB Device: Bluetooth USB Host Controller
USB Device: IR Receiver
Thunderbolt Bus:

**Comment by muchweb** [comment]
+1, latest build won't run

**Comment by stqism** [comment]
@cmacfergus @muchweb 

I introduced a regression fixing 10.7 support that caused it to crash elsewhere. It should be fixed as of now.

Please test it: https://jenkins.libtoxcore.so/job/qTox%20OS%20X/lastSuccessfulBuild/artifact/qtox.zip

**Comment by stqism** [comment]
If it launches without crashing please close this issue.

**Comment by muchweb** [comment]
@stqism, thank you. I'll ask my buddy who's using 'Apple OS' to test this tomorrow.

**Comment by cmacfergus** [comment]
Wait! Problem is not solved on my end. I just opened up qTox from the zip file, and I got the following error message:

Process:               qtox [16455]
Path:                  /Users/USER/Downloads/qtox.app/Contents/MacOS/qtox
Identifier:            im.tox.qtox
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           qtox [16455]
User ID:               501

Date/Time:             2015-04-07 13:14:29.807 -0700
OS Version:            Mac OS X 10.10.2 (14C1514)
Report Version:        11
Anonymous UUID:        905D6910-490E-A76E-1E4F-BA1F256A8456

Sleep/Wake UUID:       168846F6-0357-4CA3-A8FD-CFEBBD53FC89

Time Awake Since Boot: 9400 seconds
Time Since Wake:       220 seconds

Crashed Thread:        Unknown

Exception Type:        EXC_BAD_INSTRUCTION (SIGILL)
Exception Codes:       0x0000000000000001, 0x0000000000000000

Backtrace not available

Unknown thread crashed with X86 Thread State (64-bit):
  rax: 0x0000000000000000  rbx: 0x0000000000000002  rcx: 0x0000000001863f50  rdx: 0x00007fff7af050d0
  rdi: 0x0000000000000001  rsi: 0x0000000000000000  rbp: 0x00007fff5fbfcbc0  rsp: 0x00007fff5fbfb990
   r8: 0x00007fff5fbfcb40   r9: 0x000000000174cc46  r10: 0x00000000000117f8  r11: 0x00006080000c1478
  r12: 0x00007fff5fbfba50  r13: 0x0000000000000001  r14: 0x0000000000000000  r15: 0x0000000000000001
  rip: 0x0000000001768b94  rfl: 0x0000000000010246  cr2: 0x0000000001863f51

Logical CPU:     1
Error Code:      0x00000000
Trap Number:     6

Binary images description not available

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 18846
    thread_create: 0
    thread_set_state: 0

Model: iMac7,1, BootROM IM71.007A.B03, 2 processors, Intel Core 2 Duo, 2.8 GHz, 4 GB, SMC 1.21f4
Graphics: ATI Radeon HD 2600 Pro, ATI,RadeonHD2600, PCIe, 256 MB
Memory Module: BANK 0/DIMM0, 2 GB, DDR2 SDRAM, 667 MHz, 0x7F4F000000000000, 0x4A4D3636375153552D324720202020202020
Memory Module: BANK 1/DIMM1, 2 GB, DDR2 SDRAM, 667 MHz, 0x7F4F000000000000, 0x4A4D3636375153552D324720202020202020
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x88), Broadcom BCM43xx 1.0 (5.10.131.36.16)
Bluetooth: Version 4.3.2f6 15235, 3 services, 26 devices, 1 incoming serial ports
Network Service: AirPort, AirPort, en1
Serial ATA Device: ST3500630AS Q, 500.11 GB
Parallel ATA Device: OPTIARC  DVD RW AD-5630A
USB Device: Built-in iSight
USB Device: Cruzer
USB Device: Bluetooth USB Host Controller
USB Device: IR Receiver
Thunderbolt Bus: 

What else is going on?
